pettingzoo/test/parallel_test.py

```diff
@@ -96,11 +96,8 @@ def parallel_api_test(par_env: ParallelEnv, num_cycles=1000):
 
                 has_finished |= {
                     agent
-                    for agent, d in [
-                        (x[0], x[1] or y[1])
```

Comment:
This x[0] code is confusing, can you explain it and do some testing to make sure you’re understanding it correctly? It looks like it says (terminated false, either true) basically.

Reply:
Don't think that's necessary, the change does the same thing, may be a bit faster, and is easier to read as well.